R/AllGenerics.R

```diff
@@ -411,14 +411,21 @@ setGeneric("chromPeakData<-", function(object, value)
 #'
 #' Parameter `return.type` allows to specify the *type* of the result object.
 #' With `return.type = "Spectra"` (the default) a [Spectra] object with all
-#' matching spectra is returned. The spectra variable `"peak_id"` of the
-#' returned `Spectra` contains the ID of the chromatographic peak (i.e., the
-#' rowname of the peak in the `chromPeaks` matrix) for each spectrum.
-#' With `return.type = "Spectra"` a `List` of `Spectra` is returned. The
-#' length of the list is equal to the number of rows of `chromPeaks`. Each
-#' element of the list contains thus a `Spectra` with all spectra for one
-#' chromatographic peak (or a `Spectra` of length 0 if no spectrum was found
-#' for the respective chromatographic peak).
+#' matching spectra is returned. With `return.type = "Spectra"` a `List` of
+#' `Spectra` is returned. The length of the list is equal to the number of rows
+#' of `chromPeaks`. Each element of the list contains thus a `Spectra` with all
+#' spectra for one chromatographic peak (or a `Spectra` of length 0 if no
+#' spectrum was found for the respective chromatographic peak).
+#'
+#' Parameters `addColumnsChromPeaks` allow the user to add specific metadata
+#' columns from the chromatographic peaks (`chromPeaks`) to the returned
+#' spectra object. This can be useful to retain information such as retention
+#' time (`rt`), m/z (`mz`). The columns will be named as they is written in the
+#' `chromPeaks` object with a prefix that is defined by the parameter
+#' `addColumnsChromPeaksPrefix`. The *peak ID* (i.e., the row name of the
```

Comment:
hm. maybe we drop the `addColumnsChromPeaksPrefix` parameter for now and hard-code the prefix?

Reply:
yes it also weirdly complicates the code a bit aha. I'll remove that

Reply:
ah shoot i just re-read you message above. and you didn't want me to remove it completely... I mean it maintain that it over-complicates the code. So I would actually remove it. i'll be happy to change it back if needed later.